src/Ispell/Ispell.php

```diff
@@ -183,15 +183,16 @@ protected function parseOutput(string $output): array
                     $issues [] = $issue;
                     break;
                 case '&':
-                    $parts = explode(':', $line);
-                    $parts[0] = explode(' ', $parts[0]);
-                    $parts[1] = explode(', ', trim($parts[1]));
-                    $word = $parts[0][1];
-                    $issue = new Issue($word);
-                    $issue->line = $lineNo;
-                    $issue->offset = trim($parts[0][3]);
-                    $issue->suggestions = $parts[1];
-                    $issues [] = $issue;
+                    $matches = [];
+                    $pattern = '/^& (?<original>[^\s]+) \d+ (?<offset>\d+): (?<suggestions>.*)$/';
+                    if (1 === preg_match($pattern, $line, $matches)) {
```

Comment:
Not sure what to do if it _doesn't_ match the pattern :thinking:

Reply:
If it's not matching, previously we wouldn't have anything in `$parts`. So that would be an empty array.
I would say the line is simply ignored.

But you could write a test for that where the Ispell output doesn't match. And see if it breaks :)